.github/workflows/lint-and-test.yml

```diff
@@ -136,6 +136,23 @@ jobs:
       - name: Run clippy::pedantic for proof-of-sql-parser
         run: cargo clippy --lib -p proof-of-sql-parser -- -D clippy::pedantic
 
+  coverage:
+    name: Code Coverage
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        features: [ "test", "blitzar", "arrow", "rayon", "perf", "std", "default" ]
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+      - name: Install Dependencies
+        run: sudo apt-get update && sudo apt-get install -y clang lld
+      - uses: taiki-e/install-action@cargo-llvm-cov
+      - name: Run Tests to Generate Coverage Data
+        run: cargo llvm-cov test --workspace --features ${{ matrix.features }}
+      - name: Generate LCOV Report
+        run: cargo llvm-cov report --lcov --output-path lcov.info --fail-under-lines 90
```

Comment:
Looks like we are already over 90%, which is great to see. Can you increase this to 95% or some amount to show that it will actually fail when the coverage is too low.